docs/elasticsearch.rst

```diff
@@ -37,20 +37,14 @@ Using the Admin
 This process works both in a development environment and in production:
 
 - Open the Django admin search index list view. On Docker, this is located
-  at http://localhost:8000/admin/search/index/.
+  at http://localhost.org:8000/admin/search/index/.
 
-- Add a search index by clicking on the "Add index" button in the top right
-  corner. Optionally name it, or leave it blank to generate a valid name based
-  on the time. Spaces are not allowed in the name. Click the "SAVE" button in
-  the lower right corner.
+- On the search index list, select the checkbox next to the each index. Then
+  select "Delete selected Indexes" from the dropdown menu.
 
-- On the search index list, select the checkbox next to the newly created index
-  (the top most) and select "Populate selected search index via Celery" from
-  the Action dropdown menu. Click "Go" to start indexing asynchronously.
+- Inside the ``web`` Docker container::
 
-- In the development environment, you can watch the indexing process with::
-
-    docker-compose logs -f elasticsearch worker
+    ./manage.py reindex
```

Comment:
I'd rather use the full command so it's easy to copy-n-paste. I only ever use a bash shell when I'm debugging. I think you can just...
```suggestion
    docker-compose exec web ./manage.py reindex
```